tool/modules/uds.py

```diff
@@ -619,94 +705,140 @@ def send_key(arb_id_request, arb_id_response, level, key, timeout):
 
 def __parse_args(args):
     """Parser for module arguments"""
-    parser = argparse.ArgumentParser(prog="cc.py uds",
-                                     formatter_class=argparse.RawDescriptionHelpFormatter,
-                                     description="""Universal Diagnostic Services module for CaringCaribou""",
-                                     epilog="""Example usage:
-  cc.py uds discovery
-  cc.py uds discovery -blacklist 0x123 0x456
-  cc.py uds discovery -autoblacklist 10
-  cc.py uds services 0x733 0x633
-  cc.py uds ecu_reset 1 0x733 0x633
-  cc.py uds testerpresent 0x733
-  cc.py uds security_seed 0x3 0x1 0x733 0x633 -r 1""")
+    parser = argparse.ArgumentParser(
+                prog="cc.py uds",
+                formatter_class=argparse.RawDescriptionHelpFormatter,
+                description="Universal Diagnostic Services module for"
+                "CaringCaribou",
+                epilog="""Example usage:
+                cc.py uds discovery
+                cc.py uds discovery -blacklist 0x123 0x456
+                cc.py uds discovery -autoblacklist 10
+                cc.py uds services 0x733 0x633
+                cc.py uds ecu_reset 1 0x733 0x633
+                cc.py uds testerpresent 0x733
+                cc.py uds security_seed 0x3 0x1 0x733 0x633 -r 1""")
```

Comment:
A lot of leading whitespace has been added to every single line of this multiline string, thus messing up the terminal output.